test/WebJobs.Script.Tests/WebJobs.Script.Tests.csproj

```diff
@@ -34,6 +34,12 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
What was this change for?

Reply:
I'll undo that, not sure why VS added this.